server/opts.go

```diff
@@ -1286,6 +1286,22 @@ func (o *Options) processConfigFileLine(k string, v any, errors *[]error, warnin
 			opFiles = append(opFiles, v)
 		case []string:
 			opFiles = append(opFiles, v...)
+		case []any:
+			for _, t := range v {
+				if token, ok := t.(token); ok {
+					if v, ok := token.Value().(string); ok {
+						opFiles = append(opFiles, v)
+						continue
```

Comment:
It's a bit weird having the flow logic inverted here. Can we move the `break`s to the error cases and remove the `continue`?

Reply:
Sure. I tried to be consistent and followed the `server_tags` logic.